SimPEG/EM/Static/DC/ProblemDC.py

```diff
@@ -11,6 +11,7 @@
 import scipy as sp
 from SimPEG.Utils import Zero
 from .BoundaryUtils import getxBCyBC_CC
+from profilehooks import profile
```

Comment:
@sgkang: can we remove this for now? I am not sure we want profilehooks to be something we require for SimPEG